<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,1">

   <item row="0" column="0">
    <widget class="QGroupBox" name="move_groupBox">
     <property name="title">
      <string>ESR Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="N_pts_doubleSpinBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Number of Points</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="Navg_doubleSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Number of Averaging Runs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item row="0" column="1" colspan="3">
    <widget class="QGroupBox" name="image_groupBox">
     <property name="title">
      <string>Image Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" columnstretch="1,3">

      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="xmin_doubleSpinBox"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>X min</string>
        </property>
       </widget>
      </item>

      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="xmax_doubleSpinBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>X max</string>
        </property>
       </widget>
      </item>

      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="ymin_doubleSpinBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Y min</string>
        </property>
       </widget>
      </item>

      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="ymax_doubleSpinBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Y max</string>
        </property>
       </widget>
      </item>

      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="dx_doubleSpinBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>dx</string>
        </property>
       </widget>
      </item>

      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="dy_doubleSpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>dy</string>
        </property>
       </widget>
      </item>

      <item row="3" column="3">
       <widget class="QLabel" name="res"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Resolution</string>
        </property>
       </widget>
      </item>

      <item row="2" column="3">
       <widget class="QLabel" name="num_pts"/>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Pixel Count</string>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <item row="1" column="0" colspan="5">
    <widget class="QGroupBox" name="plot_groupBox">
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="show_pushButton">
        <property name="text">
         <string>Show</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <item row="0" column="4">
    <widget class="QGroupBox" name="camera_controls_groupBox">
     <property name="title">
      <string>Experiment Controls</string>
     </property>
     <layout class="QGridLayout" name="gridLayout" columnstretch="0,0">
      <item row="0" column="1">
       <widget class="QPushButton" name="interrupt_pushButton">
        <property name="text">
         <string>Interrupt</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="start_box">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>